src/main/twirl/uk/gov/hmrc/play/views/layouts/attorney_banner.scala.html

```diff
@@ -15,11 +15,18 @@
 *@
 @(name: Option[String], returnLinkUrl : String, returnLinkText: String)
 
-<div id="heightHolder">
-    <div class="attorneyBanner" id="attorneyBanner" data-focuses="attorneyBanner">
-        <p class="attorneyPara">
-            @Html(Messages("attorney.banner.user", name.getOrElse("Name Not Defined").toLowerCase.split(" ").map(_.toLowerCase.capitalize).mkString(" ")))
-            <a href="@returnLinkUrl" class="banner-links">@returnLinkText</a></p>
+<div class=" grid-wrapper attorneyContainer">
+    <div class="attorneyBanner grid" id="attorneyBanner">
+        <div class="grid grid-2-3">
+            <div class="inner-block">
+                @Html(Messages("attorney.banner.user", name.fold("Name Not Defined")(_.toLowerCase.split(" ").map(_.capitalize).mkString(" ").split("-").map(_.capitalize).mkString("-"))))
```

Comment:
this line is still terrible. What is it supposed to do? we are doing 'capitalize' twice, mkstring, tolowercase ....